report/Motivation/Motivation.tex

```diff
@@ -25,11 +25,18 @@ \section{Speculations on Agile's Impact}
 
 \textbf{Our highest priority is to satisfy the customer through early and continuous delivery of valuable software.}
 This principle indicates that the developers should deliver vertical slices of the software to the customer as soon as possible and continuously.
-We assume that this will allow the developers to validate the software early and learn whether the software is correct in terms of actual usage instead of in terms of specified requirements.
+A vertical slice of software refers to a functionality that is implemented across all the layers of the software - let us say a software consist of three layers: Data Management layer, Business logic layer, and User Interface layer. 
+Then a vertical slice of the authentication component could be the login part. Which would require a User interface for performing the login activity, the business logic for actually authenticating a user, and the data management for storing the users' credentials. It would not necessarily require a logout functionality.
+
+The reason we suspect this principle is about developers to deliver vertical slices is due to the wording of "valuable software". 
+We find it reasonable to assume that in order for the software to be valuable it should be functional across all layers in regards to the implemented functionality. 
+Let us, for example, take the login example again, if it is only implemented in two of the three layers it will not hold any value for the customer.
```

Comment:
Derudover kunne man godt indsætte en sætning om hvorfor det ikke vil have noget værdi.
```suggestion
Let us, for example, take the login example again. 
If it is only implemented in two of the three layers it will not hold any value for the customer.
```